pkg/pipelines/utils.go

```diff
@@ -58,6 +58,13 @@ func BootstrapRepository(o *BootstrapOptions, f clientFactory, e executor) error
 	}
 	created, _, err := client.Repositories.Create(context.Background(), ri)
 	if err != nil {
+		repo := fmt.Sprintf("%s/%s", org, repoName)
+		if org == "" {
+			repo = fmt.Sprintf("%s/%s", currentUser.Login, repoName)
+		}
+		if _, resp, err := client.Repositories.Find(context.Background(), repo); err == nil && resp.Status == 200 {
+			return fmt.Errorf("failed to create repository, repo[%s] already exists", repo)
```

Comment:
@shubhamagarwal19 The error message is being wrapped by the caller of `BootstrapRepository`, which results in a verbose error message

`failed to create the gitops repository: "https://github.com/chetan-rns/gitops.git": failed to create repository, repo[chetan-rns/gitops] already exists`

How about changing the message to something like

```suggestion
			return fmt.Errorf("repository already exists")
```

Reply:
done, Thanks !!